src/components/UTIL_FormField.component

```diff
@@ -55,7 +56,7 @@
         rendered="{!($ObjectType[sObjType].fields[field].updateable
                     || $ObjectType[sObjType].fields[field].createable && sObj['Id']=='') && $ObjectType[sObjType].fields[field].type != 'boolean'}">        
         <apex:outputLabel for="{!$Component.inputx.util_inputfield.inputx}" styleClass="slds-form-element__label" rendered="{!showLabel == true && ftype!='boolean'}" id="lblInputField">
-            <apex:outputText rendered="{!IF(required == true, true, NOT($ObjectType[sObjType].fields[field].nillable))}" id="lblInputFieldRequired">
+            <apex:outputText rendered="{!IF(overrideRequired, false, IF(required == true, true, NOT($ObjectType[sObjType].fields[field].nillable)))}" id="lblInputFieldRequired">
```

Comment:
Should the field still look required since it's still being required by the backend validation?

Reply:
Yeah I think you're right - it should still say required as it is required on the backend.  
Thanks